.appveyor.yml

```diff
@@ -90,10 +90,14 @@ install:
           Add-Content php.ini "`n extension=php_fileinfo.dll"
           Add-Content php.ini "`n extension=php_pdo_sqlite.dll"
           Add-Content php.ini "`n extension=php_sqlite3.dll"
-          # We allways include these since they are included with PHP, and it just makes the build simpler to set up
-          Add-Content php.ini "`n extension=php_pdo_mysql.dll"
-          Add-Content php.ini "`n extension=php_mysqli.dll"
-          Add-Content php.ini "`n extension=php_pdo_pgsql.dll"
+          # We always include these since they are included with PHP, and it just makes the build simpler to set up
+          if ($env:db -eq "mysql" -Or $env:db -eq "mariadb") {
```

Comment:
I was alluding to the way how driver-specific tests are disabled, e.g. `PDOPgsqlConnectionTest`:
https://github.com/doctrine/dbal/blob/41906695d6aedfc71d50df97c18a68dff902c6bd/tests/Doctrine/Tests/DBAL/Functional/Driver/PDOPgsqlConnectionTest.php#L14-L22

Reply:
I'll implement this, IMHO it's something that should be in the test anyway

```php
if ( ! $this->_conn->getDatabasePlatform() instanceof MySqlPlatform) { 
    $this->markTestSkipped('PDOMysql only test.'); 
 }